sherpa/models/basic.py

```diff
@@ -1962,7 +1962,7 @@ def calc(self, p, xlo, xhi=None, **kwargs):
 class Integrate1D(RegriddableModel1D):
 
     def __init__(self, name='integrate1d'):
-        tol = numpy.finfo(float).eps
+        tol = numpy.finfo(float).eps  # should this be DBL_EPSILON?
```

Comment:
I think this this fine. The precision of "float" depends on the machine it's run on. For Mac and Linux, it's going to be a 64 bit number, and `np.finfo(float).eps=2.220446049250313e-16`. However, if run on other architectures, it might be different. Unless I miss something, I suggest reverting the comment because that just invites the next reader to wonder why it's there.

Reply:
What I meant was that there is a module-level constant called DBL_EPSILON with the (if I remember correctly) same definition, so should this just say `tol = DBL_EPSILON`?

Reply:
I think it's the same for us, because we run on a system where the `float` equals `float64`. But on other architectures, `float` might actually equal `float32`. Not sure we'll ever make sherpa run on an old rasberry py or whatever, so it probably does not matter in practice, but in principle, the code as written gives `tol` the value that's appropriate for the particular system it's running on.